nvflare/fuel/hci/server/login.py

```diff
@@ -176,8 +176,8 @@ def handle_logout(self, conn: Connection, args: List[str]):
         conn.append_string("OK")
 
     def pre_command(self, conn: Connection, args: List[str]):
-        if args[0] in [LOGIN_CMD_NAME, CERT_LOGIN_CMD_NAME]:
-            # skip login command
+        if args[0] in [LOGIN_CMD_NAME, CERT_LOGIN_CMD_NAME, "_check_session"]:
+            # skip login and check session command
```

Comment:
what about "list_sessions" command?

We can consider make "list_sessions" and "_check_session" as constants in sess.py

For example: `CHECK_SESSION_CMD_NAME = "_check_session"`
and then we just import that constants to be used here.